test/LibraryService.test.ts

```diff
@@ -0,0 +1,49 @@
+import { initialize, Server } from '@projecttacoma/node-fhir-server-core';
```

Comment:
Forgot this comment , but let's move `LibraryService.test.ts` and `MeasureService.test.ts` to  a subdirectory `test/services/` for cleanliness